packages/ra-input-rich-text/src/RichTextInput.stories.tsx

```diff
@@ -62,155 +64,173 @@ const i18nProvider: I18nProvider = {
-                />
-                <FormInspector />
-            </SimpleForm>
+            <Card>
+                <SimpleForm
+                    defaultValues={{ body: 'Hello World' }}
+                    onSubmit={() => {}}
+                    {...props}
+                >
+                    <RichTextInput
+                        toolbar={<RichTextInputToolbar size="medium" />}
+                        label="Body"
+                        source="body"
+                    />
+                    <FormInspector />
+                </SimpleForm>
+            </Card>
         </ResourceContextProvider>
     </AdminContext>
 );
 
 export const Large = (props: Partial<SimpleFormProps>) => (
     <AdminContext i18nProvider={i18nProvider}>
         <ResourceContextProvider value="posts">
-            <SimpleForm
-                defaultValues={{ body: 'Hello World' }}
-                onSubmit={() => {}}
-                {...props}
-            >
-                <RichTextInput
-                    toolbar={<RichTextInputToolbar size="large" />}
-                    label="Body"
-                    source="body"
-                />
-                <FormInspector />
-            </SimpleForm>
+            <Card>
+                <SimpleForm
+                    defaultValues={{ body: 'Hello World' }}
+                    onSubmit={() => {}}
+                    {...props}
+                >
+                    <RichTextInput
+                        toolbar={<RichTextInputToolbar size="large" />}
+                        label="Body"
+                        source="body"
+                    />
+                    <FormInspector />
+                </SimpleForm>
+            </Card>
         </ResourceContextProvider>
     </AdminContext>
 );
 
 export const FullWidth = (props: Partial<SimpleFormProps>) => (
     <AdminContext i18nProvider={i18nProvider}>
         <ResourceContextProvider value="posts">
-            <SimpleForm
-                defaultValues={{ body: 'Hello World' }}
-                onSubmit={() => {}}
-                {...props}
-            >
-                <RichTextInput
-                    toolbar={<RichTextInputToolbar />}
-                    label="Body"
-                    source="body"
-                    fullWidth
-                />
-                <FormInspector />
-            </SimpleForm>
+            <Card>
+                <SimpleForm
+                    defaultValues={{ body: 'Hello World' }}
+                    onSubmit={() => {}}
+                    {...props}
+                >
+                    <RichTextInput
+                        toolbar={<RichTextInputToolbar />}
+                        label="Body"
+                        source="body"
+                        fullWidth
+                    />
+                    <FormInspector />
+                </SimpleForm>
+            </Card>
         </ResourceContextProvider>
     </AdminContext>
 );
```

Comment:
As we now enable fullWidth by default, can you change this story to show when it's **not** enabled?